lametro/models.py

```diff
@@ -740,8 +740,15 @@ def related_files(self):
 
         for item in agenda_items:
             for entity in item.related_entities.filter(bill__isnull=False):
-                bill_packet = BillPacket(bill=entity.bill)
-                related.extend(bill_packet.related_files)
+                try:
+                    related_bill = entity.bill
+                except LAMetroBill.DoesNotExist:
+                    # If this exception occurs, the related bill is private.
```

Comment:
let's give a little more context on why it might happen that an item has a handle on a bill but, we can't get access it.

Reply:
the could also happen if it was test item too right?